packages/react/src/time-picker/TimePicker.tsx

```diff
@@ -0,0 +1,101 @@
+import { useEffect, useState } from 'react';
+import { useCombobox } from 'downshift';
+import { type ComboboxProps } from '../combobox/Combobox';
+import { ComboboxBase } from '../combobox/ComboboxBase/ComboboxBase';
+import {
+	type DefaultComboboxOption,
+	useComboboxInputId,
+} from '../combobox/utils';
+import { type TimeFormat } from '../time-input/utils';
+import { filterOptions, generateOptions } from './utils';
+
+export function TimePicker({
+	id,
+	inputRef: inputRefProp,
```

Comment:
Why not use `forwardRef`?

Reply:
I've copied Combobox. Not sure why it was done that way, but I want the APIs to be the same

Reply:
Hmm yeah, I see where you're coming from. Although, this is inconsistent with almost all of our other components and generally with how React is used.

Reply:
29 components use `forwardRef` and three use `inputRef`. I do think we should offer `forwardRef` for every component and maybe "alias" the `inputRef` to the forwarded ref. Anyway, that's another global thing to solve for the repo. Taking a wild guess, it might be that `inputRef` was an attempt to give some context to where the ref was forwarded when there are two inputs in a component, e.g. in combobox it's an input and button?

Anywho, for another PR. I'm fine with moving forward with `inputRef`.